stanford_profile.profile

```diff
@@ -99,6 +100,15 @@ function stanford_profile_entity_field_access($operation, FieldDefinitionInterfa
   return AccessResult::neutral();
 }
 
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ */
+function stanford_profile_field_widget_options_select_form_alter(&$element, FormStateInterface $form_state, $context) {
+  if ($context['items']->getFieldDefinition()->getName() == 'layout_selection') {
```

Comment:
What are the odds that there will be another option select field with this name?

Reply:
https://git.drupalcode.org/project/layout_library/blob/8.x-1.x/layout_library.module#L34

Alot would probably break. So very unlikely

Reply:
![https://media.giphy.com/media/1HH6lJOzOXAY/giphy.gif](https://media.giphy.com/media/1HH6lJOzOXAY/giphy.gif)